ci-operator/step-registry/openshift/selfupgrade/aws/loki/openshift-selfupgrade-aws-loki-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
```

Comment:
Are you intending to configure an o/k presubmit job for this workflow in this PR? It seems a bit strange to me to see a new workflow but no consumer. ditto for other new OWNERS files added in this PR.

Reply:
yep, sorry, I forgot to mark it as WIP

Reply:
done